ROADMAP.md

```diff
@@ -126,19 +164,29 @@ for this, and I think we should not change directions here. That said, the ultim
 allowing users to create a stream and do whatever they want with it - will make it easier for
 newcomers to get started.
 
-## Improved WASM support
+### WebRTC support (browser-to-browser)
+
+| Category     | Status      | Target Completion | Tracking                                 | Dependencies                                   | Dependents |
+|--------------|-------------|-------------------|------------------------------------------|------------------------------------------------|------------|
+| Connectivity | todo        |     Q2/2023       | https://github.com/libp2p/specs/issues/475 | https://github.com/libp2p/rust-libp2p/pull/2622 https://github.com/libp2p/test-plans/issues/53 |            |
+
+
+Once WebRTC for browser-to-server is complete, we can begin work on **browser-to-browser** and complete the WebRTC connectivity story.
+The specification needs to be written and completed first.
```

Comment:
Not critical, but it's not specified here or in https://github.com/libp2p/specs/issues/475 why the browser-to-browser work will require changes in rust-libp2p.  I assume it's because there are spec additions that we'll want to accommodate, but it may be useful to add a note.  

This isn't blocking though for checking this in.